CMake/FindGonum.cmake

```diff
@@ -4,14 +4,19 @@
 if (GO_EXECUTABLE)
   execute_process(
      COMMAND ${GO_EXECUTABLE} list gonum.org/v1/gonum/mat
-     OUTPUT_VARIABLE GONUM_VERSION_STRING
+     OUTPUT_VARIABLE GONUM_RAW_STRING
      RESULT_VARIABLE RESULT
   )
   if (RESULT EQUAL 0)
-    string(REGEX REPLACE ".*([0-9]+\\.[0-9]+\\.[0-9]+[\n]+).*" "\\1"
-        GONUM_VERSION_STRING ${GONUM_VERSION_STRING})
     string(REGEX REPLACE "\n$" ""
-        GONUM_VERSION_STRING ${GONUM_VERSION_STRING})
+    GONUM_RAW_STRING ${GONUM_RAW_STRING})
+    if ("${GONUM_RAW_STRING}" STREQUAL "gonum.org/v1/gonum/mat")
+      set(GONUM_VERSION_STRING "0.0.1")
```

Comment:
It might be clearer to just do

```
set(GONUM_FOUND 1)
```

and then later in the `find_package_handle_standard_args()` call, use `GONUM_FOUND` instead of `GONUM_VERSION_STRING`. :+1:

Reply:
However, on second thought, if you only intended this to be temporary until switching to use modules, then perhaps it is not worth the change.

Reply:
I think either way we should go with `set(GONUM_FOUND 1)`, it's a small change.

Reply:
Done